lib/SILOptimizer/Mandatory/TFDifferentiation.cpp

```diff
@@ -1288,8 +1288,8 @@ class DifferentiableActivityInfo {
                 unsigned independentVariableIndex) const;
   bool isUseful(SILValue value,
                 unsigned dependentVariableIndex) const;
-  bool isVaried(SILValue value,
-                const llvm::SmallBitVector &parameterIndices) const;
+  bool isVariedByAny(SILValue value,
```

Comment:
The semantics of `isVaried` is already "is varied for any parameter". So the name need not be changed.